cmd/utils/flags.go

```diff
@@ -1469,6 +1469,10 @@ func setEthashCacheDir(ctx *cli.Context, cfg *eth.Config) {
 }
 
 func setEthash(ctx *cli.Context, cfg *eth.Config) {
+	if ctx.GlobalBool(FakePoWFlag.Name) {
```

Comment:
@ziogaschr What do you think about including this in this PR?  Although it does indeed seem like something worth doing, I'm inclined to think it should be it's own PR. You?

Reply:
Indeed I would prefer this removed from this PR. It might fit well into #243.
I will remove it now

Reply:
About the `--fakepow.poisson`, I was thinking something like `--ethash.mode` or `--ethash.powmode` enum string flag with values **fake**, **fullFake**, **poissonFake**.

Also, if we add it as a generic cli command and **not under** the `--dev.*` namespace, I would suggest adding something more fancy like `--ethash.[pow]modedangerously`.